packages/client/src/publication/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './Publication';
+export {
+  PublicationMetadataDisplayTypes,
+  PublicationContentWarning,
+  PublicationMainFocus,
+} from '../graphql/types.generated';
```

Comment:
@krzysu Not sure which enums I should export for this module -- let me know if there's a good spot to find this out!

Reply:
I guess you just have to check what the `request` argument offers for different publication queries and make it all available